common/test/views/support/cleaner/AmpEmbedCleanerTest.scala

```diff
@@ -20,30 +28,61 @@ class AmpEmbedCleanerTest extends FlatSpec with Matchers {
     document
   }
 
-  private def documentWithVideos(videoUrls: String*): Document = {
-    val doc = "<html><body>" +
-      videoUrls.map{ url: String => s"""<figure class="element-video" data-canonical-url="$url"><iframe></iframe></figure>""" }.mkString +
-      "</body></html>"
-    val document: Document = Jsoup.parse(doc)
+  //html format replaces some special characters with XMLp
```

Comment:
Hi, The method I was using to generate a HTML document, while lovely and readable, seems to be treating it as XML. I needed to remove some spurious XML encodings resulting from calling the "toString()" method for this data. 
Unfortunately the URL decoding method you linked to above does not pick these characters up, so I have to do it using .replace(). Its less than ideal, but the only way I could find to make this work.